src/components/common/Item.js

```diff
@@ -66,7 +65,39 @@ const Item = ({ id, isChildren, showPinnedOnly }) => {
   });
 
   if (isLoading || isTagsLoading || isChildrenLoading) {
-    return <Loader />;
+    switch (true) {
+      case isCollapsible: {
+        return <Skeleton variant="rect" width={'100%'} height={'56px'} />;
+      }
+      case itemType === ITEM_TYPES.FOLDER && isChildren: {
+        return null;
+      }
+      case itemType === ITEM_TYPES.FOLDER: {
+        return <Skeleton variant="rect" width={'100%'} height={'130px'} />;
+      }
+      case [
+        ITEM_TYPES.FILE,
+        ITEM_TYPES.S3_FILE,
+        ITEM_TYPES.LINK,
+        ITEM_TYPES.APP,
+      ].includes(itemType): {
+        return (
+          <Skeleton variant="rect" width={'100%'} height={SCREEN_MAX_HEIGHT} />
+        );
+      }
+      case itemType === ITEM_TYPES.DOCUMENT: {
+        return (
+          <>
```

Comment:
I think it makes sense to have a component in UI named `ItemSkeleton` so we can reuse it in builder as well. What do you think?